core/models.py

```diff
@@ -87,8 +90,10 @@ class Event(models.Model):
     main_organizer = models.ForeignKey(User, null=True, blank=True, related_name="main_organizer")
     team = models.ManyToManyField(User, blank=True)
     is_on_homepage = models.BooleanField(default=False)
+    is_deleted = models.BooleanField(default=False)
 
-    objects = EventQuerySet.as_manager()
+    objects = EventManager()
+    all_objects = models.Manager()  # This includes deleted objects
```

Comment:
Could we call this `with_deleted`? Otherwise we've got Event.objects.all() and Event.all_objects.all(), which is a bit weird :)